docs/components/movement-sensor/_index.md

```diff
@@ -612,18 +560,16 @@ readings = await my_sensor.get_readings()
 **Returns:**
 
 - (map[[string]](https://pkg.go.dev/builtin#string)interface{}): A map containing the measurements from the sensor.
-  Contents depend on sensor model and can be of any type.
+Contents depend on sensor model and can be of any type.

```

Comment:
```suggestion
    Contents depend on sensor model and can be of any type.
```
I've been indenting these to make it clear that it's meant to be part of the bullet